frontend/src/features/map/overlays/reportings/ReportingCard.tsx

```diff
@@ -106,6 +107,11 @@ export function ReportingCard({
     return `Fin dans ${Math.round(timeLeft)} h`
   }, [timeLeft, isArchived])
 
+  const isCardNotVisible = useMemo(
```

Comment:
Pour faciliter la lecture, il est en général recommandé de n'utiliser que des noms de booléens positifs (= pas de `isNot`), soit `isCardVisible` soit `isCardHidden`.

Personnellement je préfère même ne garder que du positif y compris dans le nom lui-même (ex: `is***Visible` plutôt que `is***Hidden`). Mais ça c'est plus une affaire de goût.